File: Baekjoon/1152/1152.py
    inputString = input()
    space = 0
    if len(inputString) == 0:
        print(space)
    else:
        if inputString[0] == ' ':
            space -= 1
        if inputString[-1] == ' ':
            space -= 1
        for ch in inputString:
            if ch == ' ':
                space += 1
        space += 1
        print(space)

File: Baekjoon/1152/1152.c
#include <stdio.h>
int main(void) {
		char input[1000001];
		gets(input);
		int i = 0;
		int in = 0;
		int cnt = 0;
		while (input[i] != '\0') {
				if (input[i] != ' ') {
						in = 1;
				} else if (input[i] == ' ') {
						if (in) {
								cnt++;
						}
						in = 0;
				}
				i++;
		}
		if (in) {
				cnt++;
		}
		printf("%d" , cnt);
		return 0;
}
